headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -177,14 +177,24 @@ struct statuses {
     bool power_ears;           // 0x50: STATUS_POWER_EARS
     bool scanning;             // 0x51: STATUS_SCANNING
     bool stair_spotter;        // 0x52: STATUS_STAIR_SPOTTER
-    undefined field_0x53;
+    // 0x53: Set when initally spawning a team member with the ability Pickup.
+    bool pickup_flag;
     bool grudge;       // 0x54: STATUS_GRUDGE
     bool exposed;      // 0x55: STATUS_EXPOSED (Foresight/Odor Sleuth)
     bool type_changed; // 0x56: Flag for if the monster's type has been changed
     bool boss_flag;    // 0x57: Seems to be true for boss monsters
-    undefined field_0x58;
+    // 0x58: Appears to be a flag for when a monster increasces their speed. Maybe only used
+    // by the RunLeaderTurn function to know if the leader has changed their speed stage partway
+    // through the function?
+    undefined unk_speed_boost_tracker;
```

Comment:
This name makes it look like this field is related to the Speed Boost ability, could it be renamed?